README.md

```diff
@@ -248,44 +141,35 @@ Feel free to check the [issues page](../../issues/).
 
 ## ⭐️ Show your support <a name="support"></a>
 
-> Write a message to encourage readers to support your project
-
-If you like this project...
+- We are freelance developers. If you like our work, please support us by spreading the word.
 
 <p align="right">(<a href="#readme-top">back to top</a>)</p>
 
 <!-- ACKNOWLEDGEMENTS -->
 
 ## 🙏 Acknowledgments <a name="acknowledgements"></a>
-
-> Give credit to everyone who inspired your codebase.
-
-I would like to thank...
+- This project is based on a template from Microverse full-stack web development school curriculum. You can find the template <a href="https://github.com/microverseinc/curriculum-template-databases">here</a>. Make sure you have access rights to the template.
+- I  want to thank all my clients for their patronage.
 
 <p align="right">(<a href="#readme-top">back to top</a>)</p>
 
 <!-- FAQ (optional) -->
 
-## ❓ FAQ <a name="faq"></a>
-
-> Add at least 2 questions new developers would ask when they decide to use your project.
+## ❓ FAQ (OPTIONAL) <a name="faq"></a>
+- **How can I find you?**
 
-- **[Question_1]**
+  - Please find my contact information below.
 
-  - [Answer_1]
+- **Can you give me a price range for your services?**
 
-- **[Question_2]**
-
-  - [Answer_2]
+  - To get price range, please contact us with: wabaham9@gmail.com
 
 <p align="right">(<a href="#readme-top">back to top</a>)</p>
 
 <!-- LICENSE -->
 
 ## 📝 License <a name="license"></a>
 
-This project is [MIT](./LICENSE) licensed.
-
-_NOTE: we recommend using the [MIT license](https://choosealicense.com/licenses/mit/) - you can set it up quickly by [using templates available on GitHub](https://docs.github.com/en/communities/setting-up-your-project-for-healthy-contributions/adding-a-license-to-a-repository). You can also use [any other license](https://choosealicense.com/licenses/) if you wish._
+This project is [MIT](MIT.md) licensed.
```

Comment:
Great job @wacoo 🤝

- [ ] Kindly fix the broken MIT license link to make your readme file more professional.
![mit-license](https://github.com/wacoo/vet-clinic-database/assets/63917725/c5d06845-bb6c-47b6-8b07-d5df4005b5cc)